src/site/_includes/components/LanguageList.js

```diff
@@ -16,27 +16,41 @@
 const path = require('path');
 const {getTranslatedUrls, getDefaultUrl} = require('../../_filters/urls');
 const {i18n} = require('../../_filters/i18n');
-const {languageNames} = require('../../../lib/utils/language');
+const {languageNames, scriptGroups} = require('../../../lib/utils/language');
 
 module.exports = (url, lang) => {
-  const langhrefs = getTranslatedUrls(url)
-    .filter((langhref) => langhref[0] !== lang)
-    .map((langhref) => {
-      const href = langhref[1];
-      return `<a class="w-post-signpost__link"
-        translate="no"
-        lang="${langhref[0]}"
-        href="${href}">
-      ${languageNames[langhref[0]]}</a>`;
-    });
+  let langhrefs = getTranslatedUrls(url).filter(
+    (langhref) => langhref[0] !== lang,
+  );
 
   if (langhrefs.length) {
     const enHref = path.join('/', 'i18n', 'en', getDefaultUrl(url));
-    langhrefs.push(`<a class="w-post-signpost__link"
-        translate="no"
-        lang="en"
-        href="${enHref}">
-      ${languageNames['en']}</a>`);
+    langhrefs.push(['en', enHref]);
+    langhrefs = langhrefs
+      .sort((a, b) => {
+        if (
+          scriptGroups.latin.includes(a[0]) &&
+          !scriptGroups.latin.includes(b[0])
+        ) {
+          return -1;
+        }
+      })
+      .sort((a, b) => {
+        if (
+          scriptGroups.latin.includes(a[0]) &&
+          scriptGroups.latin.includes(b[0])
+        ) {
+          return a[0].localeCompare(b[0]);
+        }
+      })
```

Comment:
I tried to get around using two `.sort()` calls here, but I couldn't think of how to combine the two into a single call. The array needed to be sorted by latin script languages (so they appear at the front) and then if the languages are indeed latin script, I wanted to sort them alphabetically. This would allow the non latin script languages to maintain their relative ordering in the list.